src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -201,7 +201,7 @@ public boolean isNameVisibleToUser(
         // Early return if user is giver
         if (question.giverType == FeedbackParticipantType.TEAMS) {
             // if response is given by team, then anyone in the team can see the response
-            if (roster.isStudentsInSameTeam(response.giver, userEmail)) {
+            if (roster.isStudentInTeam(userEmail, response.giver)) {
```

Comment:
Some context here:
- Sadly, we do not manage to migrate the database for team response. There are still entries in the database where `giver` is one of the team member's email.

Reply:
Don't close too soon. I think the fix is necessary? In addition, have you tried the legacy case where the `response.giver` is student's email?

Reply:
I don't know how to try the legacy case. I verified using the sequence described in the original issue with the uat server, and team responses did not show as anonymous.

Reply:
You can:
- Modify the current submission logic temporally to achieve that.
- Modify the instructor demo data and create a new account

Reply:
@xpdavid if we run the migration script now, would V6 still function correctly?

Reply:
Yes. Both V6 and V7 has the ability to handle both scenarios.

Reply:
I am trying to understand in the legacy case where `response.giver` is an email. There will be a bug here?

Reply:
It depends on how the legacy team name was formulated. If it's just the email of the giver then this line might return false for everyone else in the team (other than the original giver).

https://github.com/TEAMMATES/teammates/blob/faf15f9cc55f795550277b8ef7f4b8ce571ac07c/src/main/java/teammates/common/datatransfer/CourseRoster.java#L62-L72

Reply:
> It depends on how the legacy team name was formulated.

There is only one way if I am not wrong (The giver will be the email).

> If it's just the email of the giver then this line might return false 

Which is not correct? Should return `true`?

Reply:
> Which is not correct? Should return `true`?

Yeap that sounds about right. 

Given that this issue is no longer relevant (even with legacy data), any issues if I close it?